### PR TITLE
Regolith 1.0.4

### PR DESCRIPTION
**Pull request by stirante**
- Implement resolving environment variables for `exact` and `world` export types
 - Added `--update` alias flag to `--force` flag
 - Added `--timings` flag
 - Added filter settings to `when` condition data
 - Added checks for suspicious project location (blocks when in Minecraft data folders and warns when in OneDrive folders)
 - Improve downloading filters
 - Moved info about disabled subfilters to debug
 - Hidden info about nonexistent filter cache, when running from install subcommand
 - Updated go-simple-eval library to fix various bugs when evaluating `when` condition

### CODE REVIEW

**Comment by Nusiq** [comment]
Perhaps we should update the documentation:
- Mentioning new flags - `--update` and `--timings` is not necessary because it's in `--help` and we currently don't have a page in documentation that describes every flag.
- The env variable resolving from 6bb52d88da73e5b7d6bf1b419acd94350bd471e0 could be mentioned in the docs

**Comment by stirante** [comment]
Ahh right, I completely forgot about the docs

**Review by Nusiq — CHANGES_REQUESTED** [review]
Don't mark MojangDir as "suspicious path".

**Comment by Nusiq** [comment]
`go test ./...` fails on my computer on multiple tests. I'm currently checking if it's not something related to my setup.

File: regolith/compatibility_windows.go [hunk 1]
```diff
@@ -174,3 +175,47 @@ func FindPreviewDir() (string, error) {
...
+	// Check if project directory is within mojang dir
+	dir, err := FindMojangDir()
+	if err == nil && isPathWithinDirectory(path, dir) {
+		return burrito.WrappedErrorf(projectInMojangDirError, path, dir)
+	}
+	// Check if project directory is within mojang dir
+	dir, err = FindPreviewDir()
+	if err == nil && isPathWithinDirectory(path, dir) {
+		return burrito.WrappedErrorf(projectInPreviewDirError, path, dir)
+	}
```
[the remark this answers is not in the document]
Comment (stirante) [reply regolith/compatibility_windows.go:193]: How about I limit the check to `development_X_packs` folders? When I was converting a friend to regolith, previously they had a repo just for RP and so the repo was in `development_resource_packs`. Once I converted it to regolith and they pulled the changes, it started breaking.

**Review by Nusiq — APPROVED** [review]

**Pull request merged**